api/src/graphql/schema.ts

```diff
@@ -4,10 +4,13 @@ import { validationPlugin } from 'nexus-validation-plugin';
 import { applyMiddleware } from 'graphql-middleware';
 import * as types from '../modules';
 import { GQLDateBase } from './dateSchema';
+import {setup} from 'powerhouse-switchboard-plugin-demo';
+
+const demoTypes = setup();
```

Comment:
As suggested previously:
- [ ] Please provide a single separate config file where only modules are imported and setup functions are added to some array, which is the only action that needs to be done by the developer
- [ ] Please provide example of adding more data to the plugin (such as origin you've used previously, crud actions)

Reply:
Ok, got the proposable structure. See if it's ok. Apart from this typechecks just explode with errors because there's no typing as of now.